Engine/DataFeeds/LiveTradingDataFeed.cs

```diff
@@ -433,13 +433,51 @@ protected Subscription CreateSubscription(SubscriptionRequest request)
                 {
                     // this enumerator allows the exchange to pump ticks into the 'back' of the enumerator,
                     // and the time sync loop can pull aggregated trade bars off the front
-                    var aggregator = new TradeBarBuilderEnumerator(request.Configuration.Increment, request.Security.Exchange.TimeZone, _timeProvider);
-                    _exchange.SetDataHandler(request.Configuration.Symbol, data =>
+                    switch (request.Configuration.TickType)
```

Comment:
No action required, and this is probably mostly my fault :) but this type of code can get unwieldy very quickly. I believe the intent behind the `ISubscriptionEnumeratorFactory` was to remove these types of sprawling conditionals, but that change never made it's way into the live feed (mea culpa), but something to consider when refactoring in the future.